devspace.yaml

```diff
@@ -63,28 +41,24 @@ deployments:
             extended: true
         syncer:
           image: ${SYNCER_IMAGE}
+          resources:
+            limits:
```

Comment:
Not sure if we want to set these as the cpu limits can lead to the pod not getting scheduled at all and the memory limits could potentially crash the process

Reply:
Ok, I've set no limit now. I wasn't working via values, so I used:
```
dev:
  vcluster:
    resources:
      limits: ...
```